src/Models/Session.swift

```diff
@@ -20,13 +20,22 @@ public struct Session: Codable, Equatable, Sendable {
 	}
 
 	public struct InputAudioTranscription: Codable, Equatable, Sendable {
+        /* The enabled property should exist going by the events page, however fails to update if it does.
+            - https://platform.openai.com/docs/api-reference/realtime-client-events/session-update
+         
 		public var enabled: Bool
+        */
 		public var model: String
+        
+        public init(enabled: Bool = true, model: String = "whisper-1") {
+//            self.enabled = enabled
+            self.model = model
+        }
 	}
```

Comment:
I think this is an issue on the OpenAI side either with the API or docs. It says this model should have an `enabled` property, but including it causes the session update to fail. It works fine without it.